### DISCUSSION TITLE
抽签名额是不变的，这类指南是否像高考培训班一样不仅没有帮助，反而间接增加了难度？

### DISCUSSION THREAD

**Discussion by luooooob** [The-Run-Philosophy-Organization/run#140] [General]
我觉得这个仓库的目标应该是  帮助更多的人润出去

但是这个仓库的内容都是  吸引更多的人来排队

不仅没有润更多的人，反而让这条本来就拥挤的道路更挤了，这好像违背了项目的初衷？

**Comment by stackunderflow111** [comment]
普通中国人：同胞们都润了，为同胞感到高兴！
内卷中国人：同胞都润了，我就润不了呜呜呜😭😭😭

**Comment by the-run-philosophy** [comment]
并非只能去美国；而且合法留在美国的途径也不是只有h1b抽签。。

**Comment by the-run-philosophy** [comment]
欢迎阅读本站关于润美国的内容，里面有解释其他类型签证

**Comment by luooooob** [comment]
> 普通中国人：同胞们都润了，为同胞感到高兴！ 内卷中国人：同胞都润了，我就润不了呜呜呜😭😭😭

所以说这是个很现实的问题，润学总归还是要回到处理这个问题的道路上来的。

已经润了的人已经在用行动支持润学了，润学指南显然是给没润的人看的，怎么获得这些人的支持？那只能是“润学能让更多的人润出去”

“为了脱离内卷，要先卷赢别人”，这会显得很滑稽。卷h1b这种方法说了和没说一样，你就是吸引了1亿人来排队大家也知道不会多润出去一个人，反而会引起原本正在排队的人反对，得不偿失。移民中介可以搞这些东西，但是润学不能。

@the-run-philosophy 我的看法是能抽h1b的人不需要任何指南，也不需要更多的人来排队，不应该占用那么多篇幅。应该着重介绍更多不占用名额的地区和途径，并且明确表示反对大家继续卷h1b。甚至激进一点直接和卷h1b这种行为做切割，可以另开一个仓库但是不能进润学。

如果只是随便玩玩怎么搞都行，但是如果想让润学传播出圈，是一定要解决这些问题的。
